README.md

```diff
@@ -38,19 +38,34 @@ Role Variables
 --------------
 
 ```yml
-jenkins_version: "2.19.2" # The exact version of jenkins to deploy
+jenkins_version: "2.32.3" # The exact version of jenkins to deploy
 jenkins_url: "http://127.0.0.1" # The url that Jenkins will be accessible on
 jenkins_port: "8080" # The port that Jenkins will listen on
 jenkins_home: /data/jenkins # The directory on the server where the Jenkins configs will live
 
 # If you need to override any java options then do that here.
 jenkins_java_opts: "-Djenkins.install.runSetupWizard=false"
 
+# Configuration files owner and group
+jenkins_config_owner: "ubuntu"
+jenkins_config_group: "ubuntu"
+
 # The locations of the configuration files for jenkins
 jenkins_source_dir_configs: "{{ playbook_dir }}/jenkins-configs"
 jenkins_source_dir_jobs: "{{ jenkins_source_dir_configs }}/jobs"
 
-# The names of the jobs (an xml must exist in jenkins_source_dir_jobs with these names)
+# config.xml template source
+jenkins_source_config_xml: "{{ jenkins_source_dir_configs }}/config.xml"
+
+# Include custom files for jenkins installation
+jenkins_include_custom_files: false
+jenkins_custom_files: {}
+
+# Include secrets directory during installation
+jenkins_include_secrets: false
+jenkins_source_secrets: "{{ jenkins_source_dir_configs }}/secrets/"
```

Comment:
I see that the `jenkins_custom_files` will only work for files in the jenkins_home directory, if we want a "custom file" in a subdirectory it wont work. I'm guessing this is the reason we need the `jenkins_source_secrets` var.

Is it probable that other files (apart from secrets) will need to be put in subdirectories? If so, should we make the `jenkins_custom_files` able to put any file in subdirectories instead of creating new vars for each subdirectory?

If you think that there won't be any more cases where we might need to put config files in subdirectories then it's ok with me.